src/pages/projects/AllProjects.tsx

```diff
@@ -104,17 +85,15 @@ export default function AllProjects({
           <Trans>Load more</Trans>
         </div>
       ) : (
-        !isLoadingSearch &&
-        !isLoadingProjects && (
+        !isLoading && (
           <div
             className={classNames(
               'px-5 pb-5 text-center text-grey-400 dark:text-slate-200',
               concatenatedPages?.length !== 0 ? 'pt-5' : '',
             )}
           >
             {concatenatedPages?.length}{' '}
-            {concatenatedPages?.length === 1 ? t`project` : t`projects`}{' '}
-            {searchText ? t`matching "${searchText}"` : ''}
```

Comment:
since we can now use all filters at once, i've just removed the "matching..." text since there could be too many things to list.